src/pages/StatisticsChart/activities/AddressBalanceRank.tsx

```diff
@@ -143,8 +143,7 @@ export const AddressBalanceRankChart = ({ isThumbnail = false }: { isThumbnail?:
       onFetched={setStatisticAddressBalanceRanks}
       getEChartOption={getEChartOption}
       toCSV={toCSV}
-      cacheKey="AddressBalanceRank"
-      cacheMode="date"
+      queryKey="fetchStatisticAddressBalanceRank"
```

Comment:
What if define these keys as an Enum, so it's shareable won't be misspelled

Reply:
Currently, there are no plans to let them share cache across components, just use cache within the component through the key (actually, it's unnecessary, relying on the browser's HTTP cache is enough).

If it wasn't for useQuery requiring a queryKey, I would directly remove this property.

Reply:
This can be considered an accidental reuse (I knew when I was writing the code), I didn't intend to reuse it when I wrote this key, and there would be no performance issues even if it was replaced with another key.

As long as we decide not to share the cache, we don't need to actively configure the key enum for sharing the cache.
At this time, we should only configure the enum to prevent query key conflicts, but it seems unnecessary at the moment.

Reply:
It's a bit subtle because `queryKey` defined in SmartChartPage is used as a `queryKey` of useQuery, while `queryKey` shares the same session of useQuery, so they are actually reused across queries.

Say, key `avg block time` was used in `avg block time chart page`, the result is cached in useQuery Session, and can be used in `avg block time chart on homepage`.

Of course, conflicts are also should be avoided because those keys exist in the same session.

Reply:
> It's a bit subtle because `queryKey` defined in SmartChartPage is used as a `queryKey` of useQuery, while `queryKey` shares the same session of useQuery, so they are actually reused across queries.

That makes sense, this is an implementation detail that is hard to expect from the API level.

To avoid accidental key conflicts, I have now added a union key to it:
https://github.com/Magickbase/ckb-explorer-frontend/compare/7e5d6ffcb634df53e90d6df8030b0af43c1e4864..ade5a34569d57b6e460441af4d84d1ffeb5e1f59

Reply:
LGTM